tests/functional/pyocf/types/volume_cache.py

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright(c) 2022 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+from ctypes import cast, POINTER
+
+from .cache import Cache
+from .io import Io
+from pyocf.types.io import IoDir
+from .volume_exp_obj import ExpObjVolume
+
+
+class CacheVolume(ExpObjVolume):
+    def __init__(self, cache, uuid=None):
+        super().__init__(cache, uuid)
+        self.cache = cache
+
+    # TODO: remove this mock after OCF properly handles
+    # cache device flush I/O with no data set
+    def do_submit_flush(self, flush):
+        io = cast(flush, POINTER(Io))
+        io.contents._end(io, 0)
```

Comment:
Patch already on review https://github.com/Open-CAS/ocf/pull/671

Reply:
I will rebase in a separate push, to avoid mixing changes from baseline and dev branch in force-push preview

Reply:
This patch is already merged.

Reply:
ok, rebased and removed